packages/block-library/src/query-loop/index.php

```diff
@@ -81,7 +93,11 @@ function render_block_core_query_loop( $attributes, $content, $block ) {
 			)
 		)->render( array( 'dynamic' => false ) );
 	}
-	return $content;
+	return sprintf(
+		'<div %1$s>%2$s</div>',
```

Comment:
Why a hardcoded `div`? Shouldn't this be a `ul`, with each post being wrapped in a `li`? Semantically it would be a lot better that way...

Reply:
This is something I considered and can change.. I'm not opposed to this but would like more opinions

Reply:
Changed to `ul`